mlflow/store/model_registry/file_store.py

```diff
@@ -529,6 +530,7 @@ def next_version(registered_model_name):
                 return 1
 
         _validate_model_name(name)
+        source = _validate_and_resolve_source(source)
```

Comment:
@harupy If we add this validation logic to the FileStore and SqlAlchemyStore, MLflow users running on their local laptops won't be able to use MLflow models saved on their local filesystems as sources. For example, users might currently do `mlflow.sklearn.save_model(path)` and then `MlflowClient.create_model_version(..., source=path)`.